exercises/largest-series-product/introduction.md

```diff
@@ -0,0 +1,5 @@
+# Introduction
+
+You are part of a SIGINT team that has intercepted a series of encrypted communication signals from a potential terrorist group.
```

Comment:
```suggestion
You are part of a SIGINT team that has intercepted a series of encrypted communication signals from a group of bank robbers.
```

(just in case we end up triggering people who were near 9/11 or one of the other things internationally etc)

Reply:
Yeah, good call.